main/ertp/guide/amounts.md

```diff
@@ -23,12 +23,12 @@ numbers represented as BigInts. Non-fungible values may be represented as string
 particular right, or an arbitrary object representing the rights at
```

Comment:
In line 8, values don't have API methods. Also, "API methods" is not really the right term. Objects have methods, so we can say that brands have methods, and there's a brand API, but "API methods" doesn't really make sense.

Reply:
In line 9, let's say "use the `amountMath` library"

Reply:
Line 15 only partially true now. We check whether the brand looks like a brand, but that's it, whereas before, we checked whether the brand matched the brand associated with the amountMath. I would just take out `(it also checks the `brand`'s validity)`